addons/hr/report/hr_employee_badge.xml

```diff
@@ -38,7 +38,7 @@
                             <td style="width:67%" valign="center">
                                 <table style="width:155pt; height:85pt" class="table-borderless">
                                     <tr><th><div style="font-size:15pt; margin-bottom:0pt;margin-top:0pt;" align="center"><span t-out="employee.name">Marc Demo</span></div></th></tr>
-                                    <tr><td><div align="center" style="font-size:10pt;margin-bottom:5pt;"><span t-out="employee.job_id.name">Software Developer</span></div></td></tr>
+                                    <tr><td><div align="center" style="font-size:10pt;margin-bottom:5pt;"><span t-out="employee.job_id.name"/></div></td></tr>
```

Comment:
Can you do the following so that the placeholder value is still displayed inside the studio report editor ?

```suggestion
                                    <tr><td><div align="center" style="font-size:10pt;margin-bottom:5pt;"><span t-out="employee.job_id.name or ''">Software Developer</span></div></td></tr>
```